setup/artifacts/gcs.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Google Cloud Storage Artifacts"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that support reading data from artifacts can consume [Google Cloud
+Storage](https://cloud.google.com/storage/) (GCS) objects as artifacts.
+
+## Prerequisites
+
+You need a [Google Cloud Platform](https://cloud.google.com/) (GCP) project to
+host a bucket in. The next steps assume you've already [created a
+project](https://cloud.google.com/resource-manager/docs/creating-managing-projects),
+and installed [`gcloud`](https://cloud.google.com/sdk/downloads). You can check
+that `gcloud` is installed and authenticated by running:
+
+```bash
+gcloud info
+```
+
```

Comment:
Are there any APIs that need to be enabled to run the gcloud commands below? If so, you should include them here.

Reply:
GCS API is enabled by default AFAIK